core/include/utility/Exception.hpp

```diff
@@ -16,9 +16,14 @@ namespace Utility
 		Unknown_Exception          = Exception_Unknown_Exception,
         Non_existing_Image         = Exception_Non_existing_Image,
         Non_existing_Chain         = Exception_Non_existing_Chain
+        // TODO: from Chain.cpp
+        // Last image deletion ?
+        // Empty clipboard     ?
```

Comment:
In the file Chain.cpp there are some functions which are logging `Warning` messages in case of illegal actions like 

- Deletion of the last image of the chain
- Copy image from an empty clipboard

Should we turn those cases to `Exceptions` and eliminate the logging from the body of those functions?